lib/Tance.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var crypto = require('crypto');
```

Comment:
var? VAR??!?!

Reply:
Whoops. Result of copy-pasta :cirlaugh: I'll fix it.